tests/integration/tier0/check-privileges/test_user_privileges.py

```diff
@@ -0,0 +1,8 @@
+def test_check_user_privileges(shell):
+    # Create non-root user 'testuser' if not created already
+    shell("if cat /etc/passwd/ | grep -q 'testuser'; then :; else useradd testuser; fi")
```

Comment:
Could you implement the logic in python and call shell only where it is necessary